script/release/DSL-scripts/KIE_pushReleaseBranches.sh

```diff
@@ -48,12 +48,13 @@ fi
 # update kie-parent-metadata
 cd droolsjbpm-build-bootstrap/
 
-# change <version.org.uberfire>, <version.org.dashbuilder> and <version.org.jboss.errai>
+# change properties via sed as they don't update automatically
 sed -i "$!N;s/<version.org.uberfire>.*.<\/version.org.uberfire>/<version.org.uberfire>$UBERFIRE_VERSION<\/version.org.uberfire>/;P;D" pom.xml
 sed -i "$!N;s/<version.org.dashbuilder>.*.<\/version.org.dashbuilder>/<version.org.dashbuilder>$DASHBUILDER_VERSION<\/version.org.dashbuilder>/;P;D" pom.xml
 sed -i "$!N;s/<version.org.jboss.errai>.*.<\/version.org.jboss.errai>/<version.org.jboss.errai>$ERRAI_VERSION<\/version.org.jboss.errai>/;P;D" pom.xml
+sed -i "$!N;s/<latestReleasedVersionFromThisBranch>.*.<\/latestReleasedVersionFromThisBranch>/<latestReleasedVersionFromThisBranch>$RELEASE_VERSION<\/latestReleasedVersionFromThisBranch>/;P;D" pom.xml
```

Comment:
Just a technical note about sed features: You can perform multiple replacements using just one sed command (see -e in man sed). E.g. instead of 
```
sed -i replacement1 file
sed -i replacement2 file
sed -i replacement3 file
```
you can just do 
```
sed -i \
  -e replacement1 \
  -e replacement2 \
  -e replacement3 \
  file
```